.github/workflows/github-release.yml

```diff
@@ -0,0 +1,44 @@
+name: GitHub Release
+on:
```

Comment:
Maybe run this once in a while instead, like rustc nightlies? I don't know if there is a way to only run it when there was actually a change since last release though.

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onschedule

Reply:
If we're after nightlies then we probably do want a rolling 'dev' release like wasmtime...